app/models/user.py

```diff
@@ -23,7 +23,7 @@ class User(UserMixin, db.Model):
     password_hash = db.Column(db.String(_PASSWORD_HASH_CHAR_LENGTH))
 
     phone_num = db.Column(db.String(PHONE_NUM_CHAR_LENGTH))
-    dob = db.Column(db.DateTime)
+    dob = db.Column(db.DateTime, default=datetime.date(1,1,1))
```

Comment:
What specific problems do you have with it?  I think setting a birthdate to January 1st, the year 1 is functionally equivalent to having it default to unset.  The only difference is that it doesn't error because it now has the required date object.

Reply:
Setting the dob to 01/01/0001 the behind the scenes seems fine with me, but having the user see an arbitrary date within  the field seems kinda cheesy to me.